Services/FullContact.php

```diff
@@ -61,64 +61,64 @@ public function __construct($api_key)
     }
 
     /**
-     * This sets the webhook url for all requests made for this service 
-     * instance. To unset, just use setWebhookUrl(null).
-     *
-     * @author  David Boskovic <me@david.gs> @dboskovic
-     * @param   string $url
-     * @param   string $id
-     * @return  object
-     */
-    public function setWebhookUrl($url, $id = null) {
-        $this->_webhookUrl = $url;
-        $this->_webhookId  = $id;
-        return $this;
-    }
-
-    /**
```

Comment:
This removal block should be reverted. We still want to expose the ability to set the webhook parameters.